plugins/modules/database/saphana/hana_query.py

```diff
@@ -161,7 +178,10 @@ def main():
     if database is not None:
         command.extend(['-d', database])
     # -x Suppresses additional output, such as the number of selected rows in a result set.
-    command.extend(['-x', '-i', instance, '-u', user, '-p', password])
+    if userstore:
+        command.extend(['-x', '-U', user])
```

Comment:
does it mean that `instance` is not required too when `userstore=True` ? This doesn't seem to be the case in `DOCUMENTATION`, `EXAMPLES` nor `argument_spec`.

Reply:
the instance number is also used to get the file path